README.md

```diff
@@ -1089,6 +1090,17 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ---
 
+**[⬆ Back to Index](#content)**
+## MongoDB
+
+- :green_book: [MongoDB Tutorial From Scratch](https://www.mongodbtutorial.org/)
+- :bulb: [Interview Questions](https://www.interviewbit.com/mongodb-interview-questions/)
+- :bulb: [JSON Tutorial](https://www.w3schools.com/js/js_json_intro.asp)
```

Comment:
This is not related to MongoDB directly so either remove it from this section or move it to another relevant section

Reply:
@Gulshankhandale please make this change as well. It's for the JSON tutorial.